executors/http/src/main/java/net/opentsdb/query/execution/HttpQueryV3Factory.java

```diff
@@ -117,7 +119,16 @@ public QueryNodeConfig parseConfig(final ObjectMapper mapper,
   public void setupGraph(final QueryPipelineContext context, 
                          final QueryNodeConfig config,
                          final QueryPlanner planner) {
-    // No-op
+    if (((TimeSeriesDataSourceConfig) config).timeShifts() == null) {
```

Comment:
Since we're using `-timeShift` could you check the Schema factory and MockDataStore factories to verify they're the same please?